recipes/bzip2/1.0.8/conanfile.py

```diff
@@ -1,14 +1,11 @@
-#!/usr/bin/env python
-# -*- coding: utf-8 -*-
```

Comment:
Heads up. This change breaks compatibility for those using Python 2 (gasp) to run Conan. There is an en dash in the description string ("Burrows-Wheeler")

Reply:
@uilianries 
Maybe add a hook that tests that `open('conanfile.py', encoding='ascii').read()` succeeds?
(and also for the one in test_package)

Reply:
I gonna fix the description. About the hook, why not. We add as warning.

Reply:
@madebr I was going to suggest the same :)